lib/core/services/storage.service.ts

```diff
@@ -25,11 +24,18 @@ export class StorageService {
 
     private memoryStore: { [key: string]: any } = {};
     private useLocalStorage: boolean = false;
-    storagePrefix: string;
+    private _storagePrefix: string = '';
```

Comment:
This is the storage service. I think simply _prefix would be enough, since having `storageWhatever` in storage service is redundant, we could simply say `whatever`.